asio/src/examples/cpp11/fork/daemon.cpp

```diff
@@ -12,6 +12,7 @@
 #include <asio/ip/udp.hpp>
 #include <asio/signal_set.hpp>
 #include <array>
+#include <cstring>  // strerror used
```

Comment:
This PR is about adding CMake. This doesn't look like a CMake related change to me.

Reply:
There was a time in history, it doesn't compile without this!